### ISSUE TITLE
DRIVETRAIN: Jetson to CAN ODrive Protocol Implementation (w/ CAN Wrapper)

### ISSUE THREAD

**Issue by keyonjerome** [uwrobotics/uwrt_mars_rover#230]

**Comment by Nick-palmar** [comment]
@keyonjerome @ColinToft how far are we from being able to test this? Just wondering bc my plan is to try to interface nucleo <-> odrive, just to check the odrive C++ CAN interface and help you guys out when get there (this plan is assuming that jetson is not ready yet to interface with odrives through CAN yet). 

Am I correct about the jetson still needing a bit more time? If so, I should go ahead and attempt nucleo -> odrive through the C++ CAN interface to get the ball rolling?

**Comment by Nick-palmar** [comment]
@ColinToft this task is done right?

**Comment by ColinToft** [comment]
> @ColinToft this task is done right?

@Nick-palmar Yeah I feel like we will be able to close this? We have been communicating with odrive over can so I think that counts
